src/main/scala/io/sdkman/changelogs/java/OpenJdkMigrations.scala

```diff
@@ -188,4 +188,31 @@ class OpenJdkMigrations {
       .foreach { version =>
         removeVersion("java", "16.ea.15-open", version.platform)
       }
+
+  @ChangeSet(
+    order = "077",
+    id = "077-add_openjdk_java_16-loom-6",
+    author = "soberich"
+  )
+  def migrate077(implicit db: MongoDatabase): Unit =
+    Map(
+      Linux64    -> "openjdk-16-loom+6-105_linux-x64_bin.tar.gz",
+      MacOSX     -> "openjdk-16-loom+6-105_osx-x64_bin.tar.gz",
+      Windows    -> "openjdk-16-loom+6-105_windows-x64_bin.zip"
+    ).map {
+        case (platform, binary) =>
+          Version(
+            "java",
+            "16.0.6-loom", //has to be "1.2.3" e.i. numeric in order to Gradle auto detection work https://docs.gradle.org/nightly/userguide/toolchains.html#sec:auto_detection 
```

Comment:
This suffix is not currently supported on our backend yet.

Reply:
what about `16.ea.6.loom-open`?

Reply:
@eddumelendez "has to be `\d\.\d\.\d.+` at the moment.
@marc0der Could you please specify what exactly do you mean?